docs/src/main/paradox/keyspaces.md

```diff
@@ -0,0 +1,34 @@
+# Amazon Keyspaces
+
+@@@ warning
+
+**This project is not continuously tested against Amazon Keyspaces.**
+
+@@@
+
+[Amazon Keyspaces](https://aws.amazon.com/keyspaces/) is a wire compatible replacement for Cassandra.
+
+Initial testing shows that most things are working. To setup authentication follow the [Developer Guide](https://docs.aws.amazon.com/keyspaces/latest/devguide/using_java_driver.html).
+
+When using Amazon Keyspaces you need to additionally configure:
+
+```
+datastax-java-driver {
+  # QUORUM is not supported: https://docs.aws.amazon.com/keyspaces/latest/devguide/consistency.html
+  profiles.akka-persistence-cassandra-profile.basic.request.consistency = LOCAL_QUORUM
+
+  advanced {
+    metadata {
+      # not supported: https://forums.aws.amazon.com/thread.jspa?messageID=945795
+      schema.enabled = false
+      token-map.enabled = false
+    }
+    
+    # https://discuss.lightbend.com/t/akka-cluster-using-cassandra-driver-and-running-on-eks-with-keyspaces-for-cassandra-generates-weird-warning/7182
+    connection.warn-on-init-error = false
```

Comment:
I'm not sure about this one. Everything seems to be working but a warning is logged. Or is it worth a separate issue?

Reply:
Seems helpful to me